Doc/library/dataclasses.rst

```diff
@@ -221,7 +221,7 @@ Module contents
      c.mylist += [1, 2, 3]
 
    As shown above, the :const:`MISSING` value is a sentinel object used to
-   detect if the ``default`` and ``default_factory`` parameters are
+   detect if the ``default``, ``default_factory`` and `kw_only` parameters are
```

Comment:
As long as we're changing this, I think it would be an improvement to omit, in this text, the list of parameters that default to `MISSING`. Instead, we could just say something like "... used to detect if some parameters are provided by the user. This sentinel is used because ``None`` is a valid value for some parameters".

I don't think anything is gained by actually listing which parameters default to ``MISSING``, since it's clearly visible in the documentation a few lines above. Listing them just creates a future maintenance burden, as we see here.